Dockerfile

```diff
@@ -36,11 +36,13 @@ RUN pip install --trusted-host pypi.org --trusted-host files.pythonhosted.org -r
 
 # Run app with reload option
 EXPOSE 8080
-CMD if [ "${PATH_TO_OTEAPI_CORE}" != "/dev/null" ] && [ -n "${PATH_TO_OTEAPI_CORE}" ]; then pip install -U --force-reinstall -e /oteapi_core; fi && hypercorn asgi:app --bind 0.0.0.0:8080 --reload --debug --log-level debug
+CMD if [ "${PATH_TO_OTEAPI_CORE}" != "/dev/null" ] && [ -n "${PATH_TO_OTEAPI_CORE}" ]; then \
```

Comment:
Checking `PATH_TO_OTEAPI_CORE` against `/dev/null` seems quite arbitrary? Is there any reason to expect a user to set `PATH_TO_OTEAPI_CORE` to `/dev/null`?